bcdevexchange/Controllers/AboutUsController.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace bcdevexchange.Controllers

```

Comment:
I dont think we need multiple controllers. We can have all the pages as different actions within the same controller(Home Controller). If it had independent functionality or lots of business logic/routes, then it would have made sense to have a separate controller.

Reply:
Hmm that makes sense, I was modelling the page of some basic MVC projects.  I'll try fixing that.